src/app/tests/suites/certification/Test_TC_SMCO_2_5.yaml

```diff
@@ -0,0 +1,1011 @@
+    TTEST_EVENT_TRIGGER_CONTAMINATION_STATE_CLEAR:
+        type: int64u
+        defaultValue: "0xffffffff000000a6"
+    TEST_EVENT_TRIGGER_SENSITIVITY_LEVEL_HIGH:
+        type: int64u
+        defaultValue: "0xffffffff00000098"
+    TEST_EVENT_TRIGGER_SENSITIVITY_LEVEL_LOW:
+        type: int64u
+        defaultValue: "0xffffffff00000099"
+    TTEST_EVENT_TRIGGER_SENSITIVITY_LEVEL_CLEAR:
+        type: int64u
+        defaultValue: "0xffffffff000000a8"
+    EVENT_NUMBER:
+        type: int64u
+        defaultValue: 0
+
+tests:
+    - label: "Commission DUT to TH"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH subscribes to InterconnectSmokeAlarm attribute from DUT"
+      PICS: SMOKECO.S.A0008
+      command: "subscribeAttribute"
+      attribute: "InterconnectSmokeAlarm"
+      minInterval: 3
+      maxInterval: 30
+      response:
+          value: 0
+          constraints:
+              type: enum8
+
+    - label: "TH reads ExpressedState attribute from DUT"
+      PICS: SMOKECO.S.A0008 && SMOKECO.S.A0000
+      command: "readAttribute"
+      attribute: "ExpressedState"
+      response:
+          value: 0
+          constraints:
+              type: enum8
+
+    - label:
+          "TH reads TestEventTriggersEnabled attribute from General Diagnostics
+          Cluster"
+      PICS: SMOKECO.S.A0008 && DGGEN.S.A0008
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "readAttribute"
+      attribute: "TestEventTriggersEnabled"
+      response:
+          value: 1
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Interconnect Smoke Alarm Test Event"
+      PICS: SMOKECO.S.A0008 && DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_INTERCONNECT_SMOKE_ALARM
+
+    - label:
+          "TH waits for a report of InterconnectSmokeAlarm attribute from DUT
+          with a timeout of 300 seconds"
+      PICS: SMOKECO.S.A0008
+      command: "waitForReport"
+      attribute: "InterconnectSmokeAlarm"
+      timeout: 300
+      response:
+          constraints:
+              type: enum8
+              minValue: 1
+              maxValue: 2
+
+    - label: "TH reads InterconnectSmokeAlarm event from DUT"
+      PICS: SMOKECO.S.A0008 && SMOKECO.S.E08
+      command: "readEvent"
+      event: "InterconnectSmokeAlarm"
+      EVENT_NUMBER: EVENT_NUMBER + 1
+      response:
+          value: { AlarmSeverityLevel: 1 }
```

Comment:
Test plan says this should be the value reported for InterconnectSmokeAlarm, not 1.  So presumably need a saveAs there and use the value here?

Reply:
I've used saveAs before, but there seems to be a bug. I'll show you later.

Reply:
File a followup issue, drop the link here, and add a TODO comment to the YAML with the link to the issue.

Reply:
https://github.com/project-chip/connectedhomeip/issues/28096

Reply:
Put up a PR to fix that; thank you for filing.